.changeset/lucky-gifts-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend': minor
```

Comment:
Just adding things in this one right? So we can make it a `patch`

Reply:
Ah, I was trying to follow semver (i.e. `patch` is only for bugfixes) but perhaps it's different because it's still `0.y.z`? Anyway, happy to change it!

Reply:
Yep, exactly that :grin: